### ISSUE TITLE
JAWS is not announcing the correct column count with the column header

### ISSUE THREAD

**Issue by drotara** [microsoft/fluentui#17185]
### Environment Information

- **Package version(s)**: 7.153.2 +
- **Browser and OS versions**: Edge 88.0.705.50

### Describe the issue:
JAWS reads the column header but does not read the column position/count. This will help a user know how many columns there are and what column position they are in. 

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/adrotar/pen/abBqvvv

#### Actual behavior:
Reads out "{Name of Column} Column Header {Name of Column}" 

#### Expected behavior:
It should read "{Name of column }  Column Header {Name of column}, column {X} of {Y}" where X is the position of the currently selected column and Y is the total count of columns

### Documentation describing expected behavior
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7b30e4fb40-dcc1-45b4-b714-5e2c28b23ae8%7d&cid=77bec4f6-63d0-41a4-8902-fd28c44a309b

**Comment by gouttierre** [comment]
@drotara - thank you for filing this item - I saw the repro. thanks for that!

@khmakoto - Would you be able to confirm if this is a regression, or if this behavior is an issue?
